aptos-move/framework/aptos-token/sources/token_event_store.move

```diff
@@ -59,7 +59,7 @@ module aptos_token::token_event_store {
 
     #[event]
     /// Event emitted when the collection maximum is mutated
-    struct CollectionMaxiumMutate has drop, store {
+    struct CollectionMaximumMutate has drop, store {
```

Comment:
won't we need both of these though... since this is already on mainnet?

Reply:
yeah... I think I just remove the suffix blindly w/o checking the spelling...